ui/app/pages/permissions-connect/choose-account/index.scss

```diff
@@ -90,8 +140,25 @@
 
   &__bottom-buttons {
     display: flex;
-    justify-content: space-around;
-    width: 393px;
+    justify-content: space-between;
+    width: 428px;
+    margin-top: 16px;
+
+    button {
+      width: 124px;
+    }
+
+    .btn-default {
+      background: white;
+    }
+  }
+
+  &__toolip {
```

Comment:
I can't find any reference to this class

Reply:
removed in 532b5bf4d